docs/src/examples/conv_mnist_neural_ode.md

```diff
@@ -0,0 +1,427 @@
+# Convolutional Neural ODE MNIST Classifier on GPU
+
+Training a classifier for **MNIST** using a neural ordinary differential equation **NN-ODE**
+on **GPUs** with **Minibatching**.
+
+(Step-by-step description below)
+
+```julia
+using DiffEqFlux, OrdinaryDiffEq, Flux, NNlib,  Printf
+using Flux: logitcrossentropy
+using Flux.Data: DataLoader
+using MLDatasets
+using MLDataUtils:  LabelEnc, convertlabel, stratifiedobs
+using CUDA
+CUDA.allowscalar(false)
+
+function loadmnist(batchsize = bs, train_split = 0.9)
+    # Use MLDataUtils LabelEnc for natural onehot conversion
+    onehot(labels_raw) = convertlabel(LabelEnc.OneOfK, labels_raw,
+                                      LabelEnc.NativeLabels(collect(0:9)))
+    # Load MNIST
+    imgs, labels_raw = MNIST.traindata();
+    # Process images into (H,W,C,BS) batches
+    x_data = Float32.(reshape(imgs, size(imgs,1), size(imgs,2), 1, size(imgs,3)))
+    y_data = onehot(labels_raw)
+    (x_train, y_train), (x_test, y_test) = stratifiedobs((x_data, y_data),
+                                                         p = train_split)
+    return (
+        # Use Flux's DataLoader to automatically minibatch and shuffle the data
+        DataLoader(gpu.(collect.([x_train, y_train])); batchsize = batchsize,
+                   shuffle = true),
+        # Don't shuffle the test data
+        DataLoader(gpu.(collect.([x_test, y_test])); batchsize = batchsize,
+                   shuffle = false)
+    )
+end
+
+# Main
+const bs = 128
+const train_split = 0.9
+train_dataloader, test_dataloader = loadmnist(bs, train_split);
+
+down = Chain(
+             Conv((3,3),1=>64,relu,stride=1), GroupNorm(64,64),
+             Conv((4,4),64=>64,relu,stride=2,pad=1), GroupNorm(64,64),
```

Comment:
interesting. All of these Convs fit on tensor cores. I wonder if they are used.

Reply:
Tensor cores would be used with the incoming changes into the CUDNN wrappers, which haven't been merged yet